financial-connections-core/src/main/java/com/stripe/android/financialconnections/utils/InstantDebitsResultBuilder.kt

```diff
@@ -0,0 +1,31 @@
+package com.stripe.android.financialconnections.utils
+
+import android.net.Uri
+import android.util.Base64
+import androidx.annotation.RestrictTo
+import com.stripe.android.financialconnections.launcher.InstantDebitsResult
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+object InstantDebitsResultBuilder {
```

Comment:
Moves `uri ->  instantDebitsResult` building logic to `:core` to reuse it across lite and full.